- **A short introduction to Galaxy**
  - Pro:
    - very detailed (*2018-09-11 00:00:00*)
    - easy to follow (*2018-09-26 00:00:00*)
    - Very straight forward (*2018-10-08 00:00:00*)
    - The good explanation and visual help (*2018-10-08 00:00:00*)
    - step by step introduction, everything is very clear and easy to find (*2018-10-10 00:00:00*)
    - combination of text with visuals.  described in plain and simple terms. (*2018-10-11 00:00:00*)
    - easy to follow (*2018-10-15 00:00:00*)
    - clarity (*2018-10-17 00:00:00*)
    - Easy to follow and understand (*2018-10-18 00:00:00*)
    - Easy to follow with great pictures and arrows and labels (*2018-10-18 00:00:00*)
    - Detailed explanation with screenshots! (*2018-10-18 00:00:00*)
    - Intuitive and well executed. Images were useful in following the flow of the tutorial. (*2018-10-18 00:00:00*)
    - explorations, explanations (*2018-10-18 00:00:00*)
    - It was simple and straightforward. (*2018-10-23 00:00:00*)
    - Easiness,  (*2018-10-29 00:00:00*)
    - Informative, easy to follow and clear (*2018-10-30 00:00:00*)
    - This made using Galaxy very simple. (*2018-11-01 00:00:00*)
    - Simplicity (*2018-11-19 00:00:00*)
    - step-by-step instructions (*2018-12-07 00:00:00*)
    - Easy to follow (*2018-12-14 00:00:00*)
    - The step by step explanation with images (*2018-12-27 00:00:00*)
    - Very well explained (*2019-01-05 00:00:00*)
    - No ambiguity and it showed some basics.  Liked the labelled images. (*2019-01-13 00:00:00*)
    - Easy to follow and use (*2019-01-16 00:00:00*)
    - Images with arrows, and short answer questions (made sure I was on the right path). The icons in the text were also a nice touch.  (*2019-01-23 00:00:00*)
    - I liked overview of the major panels and the example walkthrough of an analysis and the screenshots accompanying them (*2019-01-28 00:00:00*)
    - Very clear. Screenshots are very helpful (*2019-02-02 00:00:00*)
    - simple and clear instructions (*2019-02-05 00:00:00*)
    - Clear and concise instructions (*2019-02-12 00:00:00*)
    - Simple and easy to follow (*2019-02-13 00:00:00*)
    - heel mooi (*2019-02-13 00:00:00*)
    - Simple guide  (*2019-02-20 00:00:00*)
    - presentation of the tutorial (*2019-03-07 00:00:00*)
    - Hands-on session is easy to follow (*2019-03-14 00:00:00*)
    - the way it presented  (*2019-03-22 00:00:00*)
    - It is simple, clear and has materials for training. (*2019-04-06 00:00:00*)
    - it was simple (*2019-04-08 00:00:00*)
    - everything (*2019-04-10 00:00:00*)
    - Tutorials are up to the point as the interface is user friendly (*2019-04-16 00:00:00*)
    - The detailed explanation (*2019-04-21 00:00:00*)
    - Very graphical and easy to follow (*2019-04-22 00:00:00*)
    - Very straight forward and convenient to catch up! (*2019-04-22 00:00:00*)
    - It was easy to follow and very clear (*2019-04-27 00:00:00*)
    - Easy to use for beginners.  (*2019-05-21 00:00:00*)
    - simple and easy to learn (*2019-05-31 00:00:00*)
    - It probably WAS helpful.  (*2019-06-05 00:00:00*)
    - almost all of it worked like the tutorial, very helpful (*2019-06-14 00:00:00*)
    - it's easy to use (*2019-06-21 00:00:00*)
    - the layout was great (*2019-06-23 00:00:00*)
    - a very clear and concise introduction. (*2019-07-28 00:00:00*)
    - I got to learn about galaxy even though its my first time   (*2019-08-05 00:00:00*)
    - Super simple and interactive (*2019-08-07 00:00:00*)
    - Clear and detailed instructions with the short clip on instructions e.g. dragging/drop files into histories. Awesome way to learn! (*2019-08-09 00:00:00*)
    - User friendly (*2019-08-14 00:00:00*)
    - Simple and clear (*2019-08-14 00:00:00*)
    - thorough (*2019-08-19 00:00:00*)
    - good explanations (*2019-08-20 00:00:00*)
    - The enogth for use in my new job (*2019-08-20 00:00:00*)
    - The step by step teaching (*2019-09-01 00:00:00*)
    - tutorial easy to follow (*2019-09-06 00:00:00*)
    - So mouch (*2019-09-06 00:00:00*)
    - The simplicity and practicality of the tutorial (*2019-09-11 00:00:00*)
    - The step-by-step guidance with explanations. (*2019-09-11 00:00:00*)
    - informative (*2019-09-12 00:00:00*)
    - The step by step (*2019-09-19 00:00:00*)
    - Everything is explained step by step. (*2019-09-24 00:00:00*)
    - EASE OF USAGE (*2019-10-10 00:00:00*)
    - The best online guide I have ever used for learning (*2019-10-17 00:00:00*)
    - very helpful visualizations for tools inputs and outputs (*2019-10-17 00:00:00*)
    - Its easy and well explained (*2019-10-22 00:00:00*)
    - great and easy to learn (*2019-10-31 00:00:00*)
    - Clarity (*2019-11-05 00:00:00*)
    - Full instructions, and it looked like it does on my version (*2019-11-07 00:00:00*)
    - graphical presentation (*2019-11-10 00:00:00*)
    - Easy to follow along (*2019-11-13 00:00:00*)
    - C'est très bien expliqué (*2019-11-19 00:00:00*)
    - very clear (*2019-11-20 00:00:00*)
    - easy to follow (*2019-11-25 00:00:00*)
    - Quick and easy to follow! (*2019-11-28 00:00:00*)
    - very good step by step instructions (*2019-11-28 00:00:00*)
    - Straightforward and easy to follow (*2019-11-28 00:00:00*)
    - the examples (*2019-12-01 00:00:00*)
    - easy to follow and very informative (*2019-12-02 00:00:00*)
    - Examples and hand-on exercises  (*2019-12-03 00:00:00*)
    - It is explained very well. It is easy to follow. (*2019-12-06 00:00:00*)
    - short and clear (*2019-12-19 00:00:00*)
    - Learning by example, actual results match data in tutorial  (*2019-12-19 00:00:00*)
    - Graphical results (*2019-12-22 00:00:00*)
    - all of the web (*2019-12-27 00:00:00*)
    - simple, informative and straightforward  (*2019-12-31 00:00:00*)
    - The questions, pictures, easy explanations, AMAZING!! (*2020-01-02 00:00:00*)
    - It is easy to understand and I hope I can start this tool easily. (*2020-01-08 00:00:00*)
    - The variety in options when it comes to software  (*2020-01-13 00:00:00*)
    - Interactive, informative and help give a sense of progress (*2020-01-14 00:00:00*)
    - detailed explanations and easy to follow (*2020-01-14 00:00:00*)
    - Very clear  (*2020-01-15 00:00:00*)
    - PhD comics (*2020-01-16 00:00:00*)
    - explained in a very simple way, user gets a nice overview about how to use galaxy (*2020-01-16 00:00:00*)
    - The step-by-step instructions were very helpful (*2020-01-22 00:00:00*)
    - clear instructions (*2020-01-22 00:00:00*)
    - Easy to follow (*2020-01-24 00:00:00*)
    - Easy to follow and helpful (*2020-01-27 00:00:00*)
    - very simple, a great introduction (*2020-01-30 00:00:00*)
    - It was simple and clearly explained  (*2020-01-30 00:00:00*)
    - It's simplicty, I will use the images for my presentation as it's much faster than the slideshow. (*2020-02-03 00:00:00*)
    - Very clear, simple and useful (*2020-02-03 00:00:00*)
    - logical flow of info (*2020-02-07 00:00:00*)
    - clearness (*2020-02-08 00:00:00*)
    - I liked the well-labeled comments/questions/hands on sections.  Great level for beginners. (*2020-02-10 00:00:00*)
    - Very Informative. Looking forward to it. Even for beginner as i am, i can using a galaxy without any problem. Thank you guys :) (*2020-02-11 00:00:00*)
    - Very easy to follow the instructions (*2020-02-18 00:00:00*)
    - everything. As fast indrotuction is really good For people like me, where we are far away of the both informatic and bioinformatic field. So the Galaxy project seems promising and I liked it. Now I just need to start to practice (*2020-02-19 00:00:00*)
    - Everything is perfectly clear. Thank's ! (*2020-02-20 00:00:00*)
    - every bit of it (*2020-02-21 00:00:00*)
    - very clear!  (*2020-02-26 00:00:00*)
    - The clear and short convenient characteristics  (*2020-03-03 00:00:00*)
    - Thank you so much for telling step by step!! This was really helpful for me. It is my first time learning about galaxy. First, I tried to take an online course, which I couldn't understand at all. However, I found this source, and I feel so grateful for this website! Thank youuu (*2020-03-08 00:00:00*)
    - Excellent! (*2020-03-10 00:00:00*)
    - very basic.   I need that (*2020-03-18 00:00:00*)
    - It's easy to understand and quiclky to learn, I love it!  (*2020-03-22 00:00:00*)
    - easy to follow, great visual aids (*2020-03-23 00:00:00*)
    - simple, informative, easy first step by step guide (*2020-03-23 00:00:00*)
    - The picture examples were very helpful. (*2020-03-24 00:00:00*)
    - Pictures with manual (*2020-03-25 00:00:00*)
    - Some videos and pics were quiet useful (*2020-03-25 00:00:00*)
    - the detailed descreption (*2020-04-04 00:00:00*)
    - Clear and easy explanation with exercise. (*2020-04-05 00:00:00*)
    - step by step approach (*2020-04-09 00:00:00*)
    - Great that a URL was provided in case of not having a dataset to analyse yet. That made it possible to get "hands on". But also nice visualizations with pictures and the video, and good examples of how to search for the tools, how the three windows are arranged and how to navigate between the different histories.  (*2020-04-15 00:00:00*)
    - Very easy to follow and good reinforcement (*2020-04-17 00:00:00*)
    - Very easy to follow and good reinforcement (*2020-04-17 00:00:00*)
    - Easy to follow, and very simple (*2020-04-20 00:00:00*)
    - It is very clear (*2020-04-22 00:00:00*)
    - It is simple and easy to follow. (*2020-04-25 00:00:00*)
    - This tutorial is so helpul ,it has all the information needed presented clearly and easy to understand. (*2020-04-26 00:00:00*)
    - yes (*2020-04-27 00:00:00*)
    - analysis of data, tools selected and overall use of the site (*2020-04-27 00:00:00*)
    - Whole tutorial and way of presentation easy to understand (*2020-05-02 00:00:00*)
    - Having comprehension questions after each part of the lessons (*2020-05-05 00:00:00*)
    - Steps were clearly written  (*2020-05-05 00:00:00*)
    - very clear instructions. (*2020-05-06 00:00:00*)
    - Yes (*2020-05-07 00:00:00*)
    - clarity of the text (*2020-05-08 00:00:00*)
    - The pictorial representation made it and easier to understand. (*2020-05-13 00:00:00*)
    - It's easy to understand. (*2020-05-15 00:00:00*)
    - questions and answers (*2020-05-15 00:00:00*)
    - Basic (*2020-05-24 00:00:00*)
    - everything works as expected (*2020-06-01 00:00:00*)
    - Well explained/detailed (*2020-06-01 00:00:00*)
    - Hands-on (*2020-06-01 00:00:00*)
    - very straightforward (*2020-06-02 00:00:00*)
    - it includes a lot of details (*2020-06-07 00:00:00*)
    - It was a very good explanation! (*2020-06-09 00:00:00*)
    - self explanatory tutorial (*2020-06-16 00:00:00*)
    - Being able to follow through. (*2020-06-19 00:00:00*)
    - Nice with hands on exercises (*2020-06-21 00:00:00*)
    - It's a very simple and objective tutorial (*2020-06-24 00:00:00*)
    - very detailed (*2020-06-25 00:00:00*)
    - It was helpful to learn to upload data. (*2020-06-25 00:00:00*)
    - easy to follow (*2020-06-29 00:00:00*)
    - Very easy to follow because of images with screen shots (*2020-07-03 00:00:00*)
  - Con:
    - the clear instruction pattern (*2018-09-09 00:00:00*)
    - Nothing (*2018-10-08 00:00:00*)
    - some buttons do not exist (anymore)? Like in "Look at all your histories" under 3. at the top line there is a "done" button - this was not displayed when I did it. That is not a tragedy but it can be confusing.... (*2018-10-10 00:00:00*)
    - speed (*2018-10-17 00:00:00*)
    - none (*2018-10-18 00:00:00*)
    - There is no answer to the question under the section entitled 'Re-run that tool with changed settings'. A 'Done' button does not appear in the view history screen as well.  (*2018-10-18 00:00:00*)
    - - (*2018-10-23 00:00:00*)
    - You could link to short description of what a fastq format is, etc... and warn that Filterbyquality sometimes is not present. I could not find it. I use FilterFASTQ instead (*2018-10-29 00:00:00*)
    - Nothing (*2018-10-30 00:00:00*)
    - This should be the first tutorial that shows up, even before the longer version as now I would feel more comfortable doing the long one whereas that one was overwhelming at first. (*2018-11-01 00:00:00*)
    - Nothing (*2018-11-19 00:00:00*)
    - everything is ok (*2019-01-05 00:00:00*)
    - I found the boxes around each of the big steps a little distracting. They are not necessary for the transition between the steps. The large headings are enough. (*2019-01-23 00:00:00*)
    - more detail about data set  (*2019-02-05 00:00:00*)
    - Nothing really,  (*2019-02-12 00:00:00*)
    - niks  (*2019-02-13 00:00:00*)
    - explain how to upload data from different sources (*2019-02-20 00:00:00*)
    -  some video lectures of the tutorial may be added  (*2019-03-07 00:00:00*)
    - it would be great if the quality score part is explained more. (*2019-03-14 00:00:00*)
    - Include in the tutorial that you have to validate your email address if you a new galaxy user who just registered. (*2019-03-18 00:00:00*)
    - If the tutorials could be downloadable, it could be a great help (*2019-04-16 00:00:00*)
    - If you make a question, include always the answer with an appropiate explanation to make everything clear. Why in the case you use the 36 filter there are so many reads discarded? Do not Forget most of people is the first time they work with this online tool.   (*2019-04-22 00:00:00*)
    - None (*2019-04-22 00:00:00*)
    - Needs to be updated. I could not follow the tutorial because the tools, the interface, etc. are different (*2019-06-05 00:00:00*)
    - the Filter by quality function was not on the program (*2019-06-14 00:00:00*)
    - none (*2019-06-21 00:00:00*)
    - will not know until I work on my own data (*2019-06-23 00:00:00*)
    - The history names in fig 1 could be showing as the next-analysis and my-analysis. (*2019-07-28 00:00:00*)
    - provide more links so we could be able to try and check if we understand galaxy certainly (*2019-08-05 00:00:00*)
    - NA -thought this was excellent and easy to follow! (*2019-08-09 00:00:00*)
    - multiple ways of importing data in the introduction (*2019-08-14 00:00:00*)
    - In this moment I dont have a suggest maybe with the practiceses (*2019-08-20 00:00:00*)
    - More publicity to make this fantastic site more popular to bioinformatic field (*2019-09-01 00:00:00*)
    - didn't felt need for more info (*2019-09-06 00:00:00*)
    - . (*2019-09-06 00:00:00*)
    - More exploration of the data analysis to aid those who are not familiar with biological data (e.g. brief explanation of FASTQ and how to interpret it) (*2019-09-11 00:00:00*)
    - nothing (*2019-09-12 00:00:00*)
    - How to import a file from my own computer and file requirements (format). (*2019-09-24 00:00:00*)
    - for some reason the filter by quality tool did not appear :( but the tutorial was fab! (*2019-10-10 00:00:00*)
    - Not that I know of (*2019-10-17 00:00:00*)
    - nothing (*2019-10-22 00:00:00*)
    - I do like youtube tutorial (*2019-10-31 00:00:00*)
    - nothing (*2019-11-13 00:00:00*)
    - some guidance on the graphs that are generated and what they mean (*2019-11-25 00:00:00*)
    - explain what the each step is doing to my data in this particular example (*2019-11-28 00:00:00*)
    - Anything to change. (*2019-12-06 00:00:00*)
    - More interpretation on result and tools used  (*2019-12-22 00:00:00*)
    - Update for updated version of Galaxy? (*2020-01-02 00:00:00*)
    - none (*2020-01-22 00:00:00*)
    - nothing (*2020-01-22 00:00:00*)
    - Maybe more tips about what's important in setting criteria to filter the data; reasons, general practices...etc.  (*2020-01-27 00:00:00*)
    - exploring a few more tools (*2020-01-30 00:00:00*)
    - Very minor issue (Maybe this is a tool versioning issue, or maybe I am not clear on instructions), but in "Hands-on: Run another tool" section, I could not find "Library to filter”. There is just "the input FASTQ file". It worked fine for me though.  (*2020-01-30 00:00:00*)
    - what types of data can be processed having what type extension? (*2020-02-07 00:00:00*)
    - a brief intro to IPython (*2020-02-08 00:00:00*)
    - There were some very slight differences between my window and the windows shown here, but honestly, I think this problem is unavoidable, and also it is good practice for people to struggle a tiny bit to figure out exactly what to click on.  So...no improvements! (*2020-02-10 00:00:00*)
    - More analysis. Like whenever we'are using a tools, and it has a data view in it in the middle viewing panel, like QC, what is the number mean by all the statistic coming. Youre the best! (*2020-02-11 00:00:00*)
    - its ok (*2020-02-19 00:00:00*)
    - it was cool, and the best I think (*2020-02-21 00:00:00*)
    - nothing (*2020-03-03 00:00:00*)
    - it's very useful (*2020-03-10 00:00:00*)
    - How do you get back to a single history showing and the other two panels (*2020-03-18 00:00:00*)
    - Typing in 'Filter by quality' does not bring up the tool you need.  (*2020-03-21 00:00:00*)
    - The indications of the tools that are used in the tutorial, for example FastQC in the server of galaxy.org it's actually quite different than what you say in the tutorial, I couldn't found the parameters.   (*2020-03-22 00:00:00*)
    - Information could be described more thoroughly. (*2020-03-24 00:00:00*)
    - Write information about the fact that in searching in Tools, the one we find is not the first shown.  (*2020-03-25 00:00:00*)
    - When you are looking smth in tools it would be better to mention, that it's not gonna be the first variant.  (*2020-03-25 00:00:00*)
    - using a power point or video to visualize the steps in galaxy  (*2020-04-04 00:00:00*)
    - Upload / Visualization of different types of files (*2020-04-06 00:00:00*)
    - adding more pictorial description of for example the exact place of the tool in the tool search, it took me long to find the tool I needed, and they didn't have that monkey-wrench image next to them (*2020-04-09 00:00:00*)
    - On the top of my head, nothing.  (*2020-04-15 00:00:00*)
    - I thought it was perfect (*2020-04-20 00:00:00*)
    - adding more movies (*2020-04-22 00:00:00*)
    - yes (*2020-04-27 00:00:00*)
    - nothing comes to mind (*2020-04-27 00:00:00*)
    - Filter by quality did not come up in the tool search  (*2020-05-05 00:00:00*)
    - nothing (*2020-05-06 00:00:00*)
    - well. I wish we could build a PC-installable Galaxy suite, similar as CCP4i-suite in crystallography field. (*2020-05-07 00:00:00*)
    - improving my knowledge about galaxy panel (*2020-05-08 00:00:00*)
    - not sure (*2020-05-15 00:00:00*)
    - Lets see whats next (*2020-05-24 00:00:00*)
    - 1> Filter by quality: this tool does not exist anymore... pls check  (*2020-05-24 00:00:00*)
    - it will be better to include more exercises (*2020-06-07 00:00:00*)
    - Some links with good explanations to what these new bioinformatic terms are for beginners . Overall, it was great, thank you! (*2020-06-19 00:00:00*)
    - More exercises  (*2020-06-21 00:00:00*)
    - Images (prints of screen) of the tool parameters configuration screen would be of great help. Only the name of parameters is placed that we must changed, but, I believe that the screen print with these parameter and de value whould much more interesting mainly for very novice users like me. (*2020-06-24 00:00:00*)
    - The tools no longer are found, that could be improved/ (*2020-06-25 00:00:00*)
    - I couldn't find "filter on quality" in the tools (*2020-07-06 00:00:00*)

- **Advanced R in Galaxy**
  - Pro:
    - Easy to follow and very usefull examples (*2020-07-03 00:00:00*)

- **From peaks to genes**
  - Pro:
    - Well structured, easy to understand. (*2018-09-17 00:00:00*)
    - pink note (*2018-09-17 00:00:00*)
    - clear procedure, good for beginners (*2018-09-17 00:00:00*)
    - well explained (*2018-09-17 00:00:00*)
    - Well-structured and helpful (*2018-09-17 00:00:00*)
    - easy to follow instructions with images (*2018-09-27 00:00:00*)
    - easy to follow instructions with images (*2018-09-27 00:00:00*)
    - simplicity (*2018-10-02 00:00:00*)
    - scope,  topic and clarity of tutorial (*2018-10-17 00:00:00*)
    - very clear and detailed instructions (*2018-10-17 00:00:00*)
    - it is clear (*2018-10-17 00:00:00*)
    - Nice clear , very informative (*2018-10-17 00:00:00*)
    - the structure that allows you to actually do the work.; clear tutorial and good help desk by walking around (*2018-10-17 00:00:00*)
    - Easy setting, sufficient help during the event of the organizers,  (*2018-10-17 00:00:00*)
    - easy to follow individual steps (*2018-11-22 00:00:00*)
    - Easy to follow tutorial (*2018-11-22 00:00:00*)
    - The tutorial. (*2018-12-20 00:00:00*)
    - Step by step introduction and repetition of features (*2019-01-08 00:00:00*)
    - Very clear instruction and step-by-step explanation (*2019-02-25 00:00:00*)
    - It is easy to follow (*2019-02-25 00:00:00*)
    - use of different tools, good explanations (*2019-02-25 00:00:00*)
    - most but if the files that need to be downloaded do not exist at UCSC, this tutorial is useless (*2019-03-05 00:00:00*)
    - short calculation times (*2019-06-11 00:00:00*)
    - I didn't have the time to like it since the intersect command is not working :( (*2019-08-15 00:00:00*)
    - not working with the bedtools intersect intervals (the other tool is currently not working) (*2019-08-21 00:00:00*)
    - The easy to follow structure of the guide plus the additional links provided in the Tips sections. (*2019-09-28 00:00:00*)
    - Nothing. (*2019-09-29 00:00:00*)
    - Clear and easy to follow (*2019-09-29 00:00:00*)
    - nic (*2019-11-28 00:00:00*)
    - The format of the tutorial is easy to follow and the instructions are clear.  The organization is such that it was easy to find where I had left off from the day before. (*2020-02-14 00:00:00*)
    - detailed explanations; step for step (*2020-02-25 00:00:00*)
    - it was easy to understand how to do it (*2020-02-25 00:00:00*)
    - Very very well designed for independent practice (*2020-02-25 00:00:00*)
    - Building more complex workflow from small steps. (*2020-04-07 00:00:00*)
    - Clarity, step by step instructions  (*2020-04-08 00:00:00*)
    - really nice explained (*2020-04-16 00:00:00*)
    - Very interesting to use genomic regions available in GEO and fetch the genes list with name from reference genome. Using Galaxy to work on genomics or transcriptomics complex datasets analysis scientifically and technically in step by step systematically do not allow to mislead to users. (*2020-05-08 00:00:00*)
    - Very little (*2020-06-11 00:00:00*)
    - everything (*2020-06-24 00:00:00*)
    - very easy to follow (*2020-06-29 00:00:00*)
  - Con:
    - maybe by adding more pictures next to the explanations? (*2018-09-17 00:00:00*)
    - More information on the flanking region tool (how it works) (*2018-09-17 00:00:00*)
    - pick a relevant induction based on your needs  (*2018-10-02 00:00:00*)
    - More visual examples (*2018-10-08 00:00:00*)
    - would like to have example outputs for important steps, so if i made a mistake in step 7 i don't have to find out in step 19 and go back to revise everything. I can just use the dummy dataset for step 19 to continue and finish the practice (*2018-10-17 00:00:00*)
    - airconditioning less cold (*2018-10-17 00:00:00*)
    - explaning more explicitly the goal of each individual exercise (*2018-10-17 00:00:00*)
    - short of time to complete the excercise (*2018-10-17 00:00:00*)
    - Chipseq data seems a little more complex than more 1-dimensional seq reads (DNA/RNAseq), so would seem to me that starting with e.g. RNAseq analysis could be more logical  (*2018-11-22 00:00:00*)
    - We did not realize the 00000000rik names were also gene names, so we spent some time thinking we did something wrong... (*2018-11-22 00:00:00*)
    - More practical excercises (*2018-12-20 00:00:00*)
    - some small exercises or quizes (*2019-02-25 00:00:00*)
    - go through the process and correct (*2019-03-05 00:00:00*)
    - I would love it to work, the tools changed in the meanwhile and it need to be updated cause it is not working anymore for some not clear reason . Most probably there is a problem with the UCSC file (I changed any kind of parameter on the interval file and i get always the same error). (*2019-08-15 00:00:00*)
    - Update (*2019-08-21 00:00:00*)
    - I recommend adding more images of the data being analysed for part 2 section Repeat workflow, 'Run a workflow with changed settings'. This would have been helpful to be able to compare my results with the guides. (*2019-09-28 00:00:00*)
    - Create peak summit file sections does not run. (*2019-09-29 00:00:00*)
    - More results illustrations (*2019-09-29 00:00:00*)
    - vse (*2019-11-28 00:00:00*)
    - in  Hands-on: Adjust chromosome names incolumn must be just number not Column:1  first part could explain more about chip seq and input data (*2020-02-12 00:00:00*)
    - I ran into technical difficulties that I cannot resolve;  it's hard to say whether this is due to the tutorial itself (visualizations hasn't worked yet;  the first analysis went fine, but the second yielded an empty data set).  Also, it took FAR longer than 3 hours; I do wonder if having a smaller mock data set would have helped this problem.    I'm trying to learn galaxy in order to incorporate an activity into my undergraduate course.  This tutorial seemed more relevant for training people who will use galaxy for their research.  A toned-down version with a smaller data set, perhaps without so many necessary file preparation steps would have been appreciated for my particular goals.  Having said that, I am guessing I am not the target audience, so I can see how this tutorial would be very useful for another audience. (*2020-02-14 00:00:00*)
    - - (*2020-02-25 00:00:00*)
    - why do i do it? (*2020-02-25 00:00:00*)
    - Hands-on: Count genes on different chromosomes - needs to be updated, instead of Column 1 only 1 should be used. Otherwise the analysis fails. (*2020-04-07 00:00:00*)
    - Incredibly hard to follow (*2020-06-11 00:00:00*)

- **Galaxy 101**
  - Pro:
    - very detailed and easy to follow, even for a complete beginner - great! (*2018-09-14 00:00:00*)
    - That all the tools had helpful explanations (*2018-09-15 00:00:00*)
    - The presentation of the tutorial and the practicality of it (*2018-09-26 00:00:00*)
    - I found it interesting and user friendly (*2018-09-27 00:00:00*)
    - It was easy to comprehend the content (*2018-09-27 00:00:00*)
    - easy to follow (*2018-09-27 00:00:00*)
    - Very easy to follow, well explained (*2018-10-31 00:00:00*)
    - Easy to follow (*2018-11-28 00:00:00*)
    - Easy to follow, but the task is not trivial! (*2018-11-28 00:00:00*)
    - Very informative, and nicelly (*2018-12-07 00:00:00*)
    - Very good explanations, good demonstration of Galaxy potential for begginer  (*2019-01-31 00:00:00*)
    - Screenshots showing which options should be selected  (*2019-02-04 00:00:00*)
    - The step are easy to follow  (*2019-02-10 00:00:00*)
    - Very simple to follow and learn on my own (*2019-02-25 00:00:00*)
    - Very straight-forward. Easy reading, easy learning. Gives a great first contact with the software. (*2019-03-13 00:00:00*)
    - The step by step demonstration was really helpful (*2019-04-03 00:00:00*)
    - I liked being able to compare my screenshots and felt the directions were generally very good. Definitely have a better handle on how to use the program, especially re-using workflows. (*2019-04-25 00:00:00*)
    - explanations were very clear ! (*2019-04-25 00:00:00*)
    - Instructive (*2019-05-01 00:00:00*)
    - # No results returned from query. (*2019-05-15 00:00:00*)
    - Ease of understanding (*2019-05-16 00:00:00*)
    - Very detailed description of every step (*2019-05-29 00:00:00*)
    - The instructions were very clear and easy to follow (*2019-05-30 00:00:00*)
    - Nice combination of presentations and tutorial, clear steps including tips and comments (*2019-06-11 00:00:00*)
    - Good introduction to Galaxy. I like the worklflow function the most.  (*2019-06-11 00:00:00*)
    - Good speed, detailed explanation for the introduction (*2019-06-11 00:00:00*)
    - It explains really clearly. (*2019-06-16 00:00:00*)
    - its very helpful to get a basic idea of how the platform works (*2019-07-06 00:00:00*)
    - I liked how easily it flows, some concepts of biology are briefly explained, it helps a lot!  (*2019-08-02 00:00:00*)
    - Screenshots and clear colour coding made it extremely easy to follow (*2019-08-28 00:00:00*)
    - It was very clear and intuitive  (*2019-08-28 00:00:00*)
    - it was sufficient (*2019-08-28 00:00:00*)
    - Easy to follow and useful (*2019-08-28 00:00:00*)
    - Very clear (*2019-08-28 00:00:00*)
    - Very easy to follow (*2019-09-06 00:00:00*)
    - very detailed (*2019-09-10 00:00:00*)
    - It was really easy to follow and interesting to learn (*2019-09-12 00:00:00*)
    - Easy to follow (*2019-10-07 00:00:00*)
    - details (*2019-10-08 00:00:00*)
    - Useful and easy to follow (*2019-10-09 00:00:00*)
    - It was easy to follow (*2019-10-14 00:00:00*)
    - Informative and easy to follow (*2019-10-16 00:00:00*)
    - Step by step, clear, accurate (*2019-11-01 00:00:00*)
    - Easy  (*2019-11-21 00:00:00*)
    - very clear instructions (*2019-11-29 00:00:00*)
    - The flexibility and power of workflows (*2019-12-05 00:00:00*)
    - Clear step-by-step illustrations with potential issues (*2019-12-05 00:00:00*)
    - In general, it is good. For first time user, some instruction should include more details. (*2019-12-05 00:00:00*)
    - Friendly workflow (*2019-12-05 00:00:00*)
    - Repeatable workflow for fresh man to this server. (*2020-01-02 00:00:00*)
    - The fact that it had a task in mind (*2020-01-28 00:00:00*)
    - Very organised open source to find out important tools for work. (*2020-02-04 00:00:00*)
    - I appreciated that this tutorial used a smaller data set, so that the analysis went much faster.  I also appreciated comments on what to look for if the analysis didn't work.  I am a genetics instructor at a PUI, so this tutorial was at a good level for me to think about what I want to teach my students. (*2020-02-14 00:00:00*)
    - step by step teach me how to know how to use. (*2020-02-24 00:00:00*)
    - - (*2020-03-19 00:00:00*)
    - very easy to follow (*2020-04-15 00:00:00*)
    - Easy to follow. The comments were very helpfull (*2020-06-29 00:00:00*)
  - Con:
    - If there was a guide to say which tool to use for a specific analysis to be made. Unless I know which tool to look for, I'd be lost (*2018-09-15 00:00:00*)
    - The rate of update of the tutorial (*2018-09-26 00:00:00*)
    - nothing for now (*2018-09-27 00:00:00*)
    - Nothing (*2018-10-31 00:00:00*)
    - One could add a section describing the use of multiple datasets, tags, etc. (*2018-11-04 00:00:00*)
    - Including some assignment or home work (*2018-11-28 00:00:00*)
    - update some of the options and screenshots to be consistent with the latest version of Galaxy (*2018-11-28 00:00:00*)
    - The explanation about the use workflow on Repeats (*2018-12-07 00:00:00*)
    - Galaxy at usegalaxy.org ends up showing "the white screen of death" extremely often (10s of times during following this tutorial). (*2019-01-14 00:00:00*)
    - Should provide practice every step (*2019-02-10 00:00:00*)
    - Fatal error: Exit code 1 () Error running sorter.py Command '(grep '^#' /galaxy-repl/main/files/030/998/dataset_30998511.dat) >> /galaxy-repl/main/files/030/998/dataset_30998729.dat' returned non-zero exit status 1 (*2019-04-18 00:00:00*)
    - Some of the instructions are outdated as newer versions of data in UCSC main have been added, which makes some parts slightly confusing (*2019-04-25 00:00:00*)
    - this is a bit too long (*2019-04-25 00:00:00*)
    - I do not know - I know only Biopython (*2019-05-01 00:00:00*)
    - # No results returned from query. (*2019-05-15 00:00:00*)
    - Adding more examples of its applications and how it is used in real time (*2019-05-16 00:00:00*)
    - I couldn't find a 'display at UCSC browser' option when I get to this step (*2019-05-29 00:00:00*)
    - It would be nice if there is more explanations on the significance of each step and what exactly is being performed when selecting different tools. (*2019-05-30 00:00:00*)
    - Couldn´t really see what the results in the genome browser would tell me and how I apply this for my work.  (*2019-06-11 00:00:00*)
    - by giving more free tutorials (*2019-07-06 00:00:00*)
    - Maybe add more examples, more explanations on the datasets used.  (*2019-08-02 00:00:00*)
    - More clarity in instructions for the solution to the question; "For the first 3 exons in your file, what is the number of SNPs that fall into that exon?" The description of the solution steps was unclear. (*2019-08-09 00:00:00*)
    - More clarity in instructions for the solution to the question; "For the first 3 exons in your file, what is the number of SNPs that fall into that exon?" The description of the solution steps was unclear. (*2019-08-09 00:00:00*)
    - Nothing - I thought it was great! :) (*2019-08-28 00:00:00*)
    - Nothing (*2019-08-28 00:00:00*)
    - the tiny mistakes (*2019-08-28 00:00:00*)
    - Ability to zoom on the graphics (*2019-09-06 00:00:00*)
    - none (*2019-09-10 00:00:00*)
    - Its good (*2019-09-12 00:00:00*)
    - probably covers the basics pretty well but would like more personally (*2019-10-07 00:00:00*)
    - other cases (*2019-10-08 00:00:00*)
    - nothing (*2019-10-14 00:00:00*)
    - The display is too wide when you want to juxtapose the tutorial window with galaxy (*2019-11-01 00:00:00*)
    - Maybe automattically update to gencode updates, there is no genecode v29 anymore (*2019-11-21 00:00:00*)
    - The workflow could be simpler, such as directly count the number of SNPs for each exon, and show the counts in descending order, to get the results in one step (*2019-12-05 00:00:00*)
    - Some guides are not clear. (*2019-12-05 00:00:00*)
    - Explanations and meanings for this activity (*2019-12-05 00:00:00*)
    - Further clarification to the function input parameter maybe need. (*2020-01-02 00:00:00*)
    - A bit more clarity on how to correctly indicate the column names would have saved me a lot of time trying different variations when "Column: 4" was not recognized as an input  (*2020-01-28 00:00:00*)
    - I don't think so (*2020-02-04 00:00:00*)
    - Commands can be little more detailed (*2020-02-04 00:00:00*)
    - At the section on "recovering exon info" it would be nice to prompt the audience to consider which data sets they should compare, and which columns should be used (maybe as a side activity).  When viewing results at UCSC main, ideally it would be nice to direct audience something in particular to look for. There is a lot of data, which will definitely overwhelm my students.  This is a meta suggestion....I'm going through the introductory tutorials one by one and found this one to be more introductory than the "peaks to genes" tutorial.  I wonder if considering reordering tutorials (make this one the second on the list?) or naming tutorials would help audience.    Great tutorial - I definitely appreciate your work on this! (*2020-02-14 00:00:00*)
    - realize Galaxy and know what features in The Galaxy (*2020-02-24 00:00:00*)
    - Column:4 is not working. It should be changed to "4" only. (*2020-03-04 00:00:00*)
    - Explanation how to use UCSC browser to see user track (*2020-03-12 00:00:00*)
    - - (*2020-03-19 00:00:00*)
    - failed at step of either join or count. column drop down was not available in count, so entered manually. when count applied, error on column sepecification (*2020-03-30 00:00:00*)
    - Everything is great (*2020-05-22 00:00:00*)

- **Galaxy 101 for everyone**
  - Pro:
    - The hands on exercises (*2019-11-11 00:00:00*)
    - It was more advanced than the other introductory tutorial I had read. (*2020-02-07 00:00:00*)
    - Learn how to run statistics (*2020-03-16 00:00:00*)
    - extensive, simple, step by step (*2020-03-23 00:00:00*)
    - Error when I followed instructions (*2020-03-26 00:00:00*)
    - Error when I followed instructions (*2020-03-26 00:00:00*)
    - step step guide tips (*2020-05-19 00:00:00*)
    - Clarity, organization and content (*2020-06-15 00:00:00*)
    - Clear directions for execution and correlation of results. (*2020-06-22 00:00:00*)
    - Responsiveness of contact persons. (*2020-06-23 00:00:00*)
    - Creating Workflow (*2020-06-24 00:00:00*)
    - Very easy to follow tutorial. With usefull tips (*2020-06-30 00:00:00*)
    - Easy to follow (*2020-06-30 00:00:00*)
  - Con:
    - The images about the diamonds are not very legible (*2019-11-11 00:00:00*)
    - introduce the tutorial with a statement of what type of data is being processed. Only when I saw a picture did I realize that your processing a flower called iris. (*2020-02-07 00:00:00*)
    - in section "Visualize Iris dataset with Scatterplot w ggplot2"  using the Data point options as user defined point options (relative size of points”: 2.0") originates error. Leaving default parameters corrects error.  (*2020-03-23 00:00:00*)
    - When I set “Data point options”: User defined point options “relative size of points” to 2.0, I got an error "Error in alpha * 255 : non-numeric argument to binary operator Calls: ggsave ... validGP -> numnotnull -> alpha -> <Anonymous> -> encode_c" (*2020-03-26 00:00:00*)
    - When I set “Data point options”: User defined point options “relative size of points” to 2.0, I got an error "Error in alpha * 255 : non-numeric argument to binary operator Calls: ggsave ... validGP -> numnotnull -> alpha -> <Anonymous> -> encode_c" (*2020-03-26 00:00:00*)
    - tool search tips (*2020-05-19 00:00:00*)
    - PRE-WORKSHOP/PREPARATORY MATERIALS (*2020-06-23 00:00:00*)
    - It is perfect (*2020-06-24 00:00:00*)
    - nothing (*2020-06-30 00:00:00*)

- **IGV Introduction**
  - Pro:
    - That there was a tutorial to begin with!   Thank you! (*2019-06-05 00:00:00*)
    - step by step instructions for students (*2019-10-16 00:00:00*)
    - It is easy to follow and summurazing many tools features as much as possible. (*2020-02-12 00:00:00*)
  - Con:
    - Actually show some use-cases (there is only one and i am unable to load ENCODE data) (*2019-01-15 00:00:00*)
    - Cannot find C2C12 file (*2019-02-25 00:00:00*)
    - One cannot explain an unknown by substituting another unknown.   Therefore do not use abbreviations such as BAM, BED, MAF, VCF etc.  Consider incorporating a complete or partial glossary (*2019-06-05 00:00:00*)
    - RNASeq of C2C12 not available (*2019-08-22 00:00:00*)
    - C2C12 no longer exists.  Also the Sashimi plot does not look like the curved image shown and thus confusing to the student. (*2019-10-16 00:00:00*)
    - A real-life and frequently performed  case tutorial could be shown for demonstrating what could be done in IGV. (*2020-02-12 00:00:00*)

- **Introduction to Genomics and Galaxy**
  - Pro:
    - very detailed and easy to follow, thank you (*2018-09-15 00:00:00*)
    - Broke down the concepts well. Introduced ideas one  step at a time with good explanations that minimized jargon. And worked to make concepts tangible with hands on use of Galaxy. (*2019-02-28 00:00:00*)
    - very clear and easy to follow (*2019-04-09 00:00:00*)
    - It is hands-on approach (*2019-10-25 00:00:00*)
    - It was very easy to follow. (*2019-10-31 00:00:00*)
    - I really liked how this tutorial encourages its audience to try to figure out which tools to use and how to use them to address the question. (*2020-02-20 00:00:00*)
    - Steps are clearly explained (*2020-03-17 00:00:00*)
    - Verifiable step-by-step (*2020-04-13 00:00:00*)
    - I like the brief but detailed and easy to understand instructions with explanations (*2020-06-11 00:00:00*)
    - Very clear (*2020-06-12 00:00:00*)
    - Easy to follow (*2020-06-30 00:00:00*)
  - Con:
    - Galaxy says it has ran the workflow, but nothing happens. (*2019-01-16 00:00:00*)
    - It could be better to put the overlapped extron images here too (*2019-04-09 00:00:00*)
    - intersect never worked (*2019-08-22 00:00:00*)
    - Update the intersect phase since that tool is no more available (*2019-08-23 00:00:00*)
    - the intersect tool gives an infinite loop.  The annotations changed on UCSC,  "table: Comprehensive (wgEncodeGencodeCompV24" instead of "table: known genes" (*2019-09-24 00:00:00*)
    - Make it a bit more in-depth (*2019-10-25 00:00:00*)
    - this .bed file contain more row than number of genes on chr22 ?! (*2020-02-15 00:00:00*)
    - Just before the section on "Examine the data" there is a [TODO] line that has been left in.  Oops! (*2020-02-20 00:00:00*)
    - In the overlapping gene illustration, the appearance of overlapping genes on both strands is not as clear as the other illustrations.  (*2020-04-13 00:00:00*)
    - the section on Visualize the overlapping genes could do with some improvement. I struggled to understand what scale track was and how to click it at the begining but eventually got around it. Maybe other users found it easy but that was my challenge. (*2020-06-11 00:00:00*)
    - Slightly outdated, some tool locations seem to have changed (*2020-06-12 00:00:00*)
    - Nowadays the tool Intersect, it's no longer in the Operate on Genomic Intervals toolbox. (*2020-06-30 00:00:00*)

- **NGS data logistics**
  - Pro:
    - good overview over method and files (*2018-09-16 00:00:00*)
    - good overview over method and files (*2018-09-16 00:00:00*)
    - good overview over method and files (*2018-09-16 00:00:00*)
    - makes life easy (*2018-09-27 00:00:00*)
    - a very well written tutorial with detailed information for a person learning Chip-Seq from scartch  (*2018-11-03 00:00:00*)
    - Clear and consise (*2018-11-17 00:00:00*)
    - The explanation is very clear and concise, thanks! (*2018-11-27 00:00:00*)
    - Explanation of theoretical basics. Links to additional resources. (*2019-01-17 00:00:00*)
    - awesome and answered many of the questions in my head (*2019-01-22 00:00:00*)
    - I liked the videos. (*2019-04-30 00:00:00*)
    - the explanation part (*2019-05-23 00:00:00*)
    - all of it (*2019-05-24 00:00:00*)
    - Anyone could have understood what it has been explained!! (*2019-06-03 00:00:00*)
  - Con:
    - not as detailed and easy to follow as the first tutorials, examples not well shown (*2018-09-16 00:00:00*)
    - not as detailed and easy to follow as the first tutorials, examples not well shown (*2018-09-16 00:00:00*)
    - not as detailed and easy to follow as the first tutorials, examples not well shown (*2018-09-16 00:00:00*)
    - very good (*2018-09-27 00:00:00*)
    - I don't see the files name as sample1-r.fq.gz to sample 4 to download and do that practically. However, I do understand the video and it was very helpful (*2018-11-03 00:00:00*)
    - Mark Duplicates step fails:  Fatal error: Exit code 1 () Picked up _JAVA_OPTIONS: -Djava.io.tmpdir=/galaxy-repl/main/jobdir/022/055/22055178/_job_tmp -Xmx7g  -Xms256m 11:37:37.876 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/cvmfs/main.galaxyproject.org/deps/_conda/pkgs/picard-2.18.2-py36_0/share/picard-2.18.2-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so [Thu Jan 17 11:37:37 CST 2019] MarkDuplicates INPUT=[Filter_on_data_7__Filtered_BAM] OUTPUT=/galaxy-repl/main/files/029/341/dataset_29341054.dat METRICS_FILE=/galaxy-repl/main/files/029/341/dataset_29341053.dat REMOVE_DUPLICATES=true ASSUME_SORTED=true DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES OPTICAL_DUPLICATE_PIXEL_DISTANCE=100    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true ADD_PG_TAG_TO_READS=true PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false [Thu Jan 17 11:37:37 CST 2019] Executing as g2main@roundup55.tacc.utexas.edu on Linux 3.10.0-693.21.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.2-SNAPSHOT INFO 2019-01-17 11:37:38 MarkDuplicates Start of doWork freeMemory: 248936480; totalMemory: 259588096; maxMemory: 7265714176 INFO 2019-01-17 11:37:38 MarkDuplicates Reading input file and constructing read end information. INFO 2019-01-17 11:37:38 MarkDuplicates Will retain up to 26325051 data points before spilling to disk. [Thu Jan 17 11:37:38 CST 2019] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 0.02 minutes. Runtime.totalMemory()=382373888 To get help, see http://broadinstitute.github.io/picard/index.html#GettingHelp Exception in thread "main" htsjdk.samtools.SAMFormatException: SAM validation error: ERROR: Record 262, Read name M01368:16:000000000-A3M99:1:2114:3862:13382, bin field of BAM record does not equal value computed based on alignment start and end, and length of sequence to which read is aligned  at htsjdk.samtools.SAMUtils.processValidationErrors(SAMUtils.java:454)  at htsjdk.samtools.BAMFileReader$BAMFileIterator.advance(BAMFileReader.java:812)  at htsjdk.samtools.BAMFileReader$BAMFileIterator.next(BAMFileReader.java:797)  at htsjdk.samtools.BAMFileReader$BAMFileIterator.next(BAMFileReader.java:765)  at htsjdk.samtools.SamReader$AssertingIterator.next(SamReader.java:576)  at htsjdk.samtools.SamReader$AssertingIterator.next(SamReader.java:548)  at picard.sam.markduplicates.MarkDuplicates.buildSortedReadEndLists(MarkDuplicates.java:495)  at picard.sam.markduplicates.MarkDuplicates.doWork(MarkDuplicates.java:232)  at picard.cmdline.CommandLineProgram.instanceMain(CommandLineProgram.java:282)  at picard.cmdline.PicardCommandLine.instanceMain(PicardCommandLine.java:98)  at picard.cmdline.PicardCommandLine.main(PicardCommandLine.java:108) (*2019-01-17 00:00:00*)
    - nothing I can think of (*2019-01-22 00:00:00*)
    - I would suggest to make the videos longer and theory shorter. (*2019-04-30 00:00:00*)
    - it is nice (*2019-05-23 00:00:00*)
    - add example with bowtie2  (*2019-05-24 00:00:00*)
    - Please date this content.  Is it out of date?  It does not mention kallisto, sleuth or salmon.  Are these available in a cloud somewhere?  Are they better? (*2019-08-27 00:00:00*)
    - You could make the tutorial with the same files (*2019-10-28 00:00:00*)
    - The tutorial is very recipe-like but does not tell you what recipe you are making (so why you use particular options and not others) or explain how to modify recipe for your purposes. As such useful but of limited duse (*2020-02-09 00:00:00*)
    - Video are difficult to follow  (*2020-03-24 00:00:00*)

- **R basics in Galaxy**
  - Pro:
    - Easy to follow (*2019-10-09 00:00:00*)
    - Very thorough (*2019-10-09 00:00:00*)
    - Very nice. Very good and easy to get used to R logic in applicated way. (*2020-01-13 00:00:00*)
    - The instructions were clear and neat. Also, it covers important aspects of the language. (*2020-02-12 00:00:00*)
    - It was easy to follow (*2020-04-03 00:00:00*)
    - Thank you so much for providing the related material for R (*2020-06-07 00:00:00*)
    - Good guide, with usefull examples  (*2020-06-30 00:00:00*)
  - Con:
    - Some variable names are not copied right in a few hands-on sections (ie chromosomes_2 and snp_chromosomes_2 under coerce values) (*2019-10-09 00:00:00*)
    - Maybe finish by a biological application like "Output gene that are longer than 100000bp". Another idea can to ask a question in a way that you have to use all function seen in one. Like a short short application project.  (*2020-01-13 00:00:00*)
    - There should be one task that push us to combine and use the learned knowledge. (*2020-02-12 00:00:00*)

