- **A short introduction to Galaxy**
  - Pro:
    - very detailed (*11/09/2018*)
    - easy to follow (*26/09/2018*)
    - Very straight forward (*08/10/2018*)
    - The good explanation and visual help (*08/10/2018*)
    - step by step introduction, everything is very clear and easy to find (*10/10/2018*)
    - combination of text with visuals.  described in plain and simple terms. (*11/10/2018*)
    - easy to follow (*15/10/2018*)
    - clarity (*17/10/2018*)
    - Easy to follow and understand (*18/10/2018*)
    - Easy to follow with great pictures and arrows and labels (*18/10/2018*)
    - Detailed explanation with screenshots! (*18/10/2018*)
    - Intuitive and well executed. Images were useful in following the flow of the tutorial. (*18/10/2018*)
    - explorations, explanations (*18/10/2018*)
    - It was simple and straightforward. (*23/10/2018*)
    - Easiness,  (*29/10/2018*)
    - Informative, easy to follow and clear (*30/10/2018*)
    - This made using Galaxy very simple. (*01/11/2018*)
    - Simplicity (*19/11/2018*)
    - step-by-step instructions (*07/12/2018*)
    - Easy to follow (*14/12/2018*)
    - The step by step explanation with images (*27/12/2018*)
    - Very well explained (*05/01/2019*)
    - No ambiguity and it showed some basics.  Liked the labelled images. (*13/01/2019*)
    - Easy to follow and use (*16/01/2019*)
    - Images with arrows, and short answer questions (made sure I was on the right path). The icons in the text were also a nice touch.  (*23/01/2019*)
    - I liked overview of the major panels and the example walkthrough of an analysis and the screenshots accompanying them (*28/01/2019*)
    - Very clear. Screenshots are very helpful (*02/02/2019*)
    - simple and clear instructions (*05/02/2019*)
    - Clear and concise instructions (*12/02/2019*)
    - Simple and easy to follow (*13/02/2019*)
    - heel mooi (*13/02/2019*)
    - Simple guide  (*20/02/2019*)
    - presentation of the tutorial (*07/03/2019*)
    - Hands-on session is easy to follow (*14/03/2019*)
    - the way it presented  (*22/03/2019*)
    - It is simple, clear and has materials for training. (*06/04/2019*)
    - it was simple (*08/04/2019*)
    - everything (*10/04/2019*)
    - Tutorials are up to the point as the interface is user friendly (*16/04/2019*)
    - The detailed explanation (*21/04/2019*)
    - Very graphical and easy to follow (*22/04/2019*)
    - Very straight forward and convenient to catch up! (*22/04/2019*)
    - It was easy to follow and very clear (*27/04/2019*)
    - Easy to use for beginners.  (*21/05/2019*)
    - simple and easy to learn (*31/05/2019*)
    - It probably WAS helpful.  (*05/06/2019*)
    - almost all of it worked like the tutorial, very helpful (*14/06/2019*)
    - it's easy to use (*21/06/2019*)
    - the layout was great (*23/06/2019*)
    - a very clear and concise introduction. (*28/07/2019*)
    - I got to learn about galaxy even though its my first time   (*05/08/2019*)
    - Super simple and interactive (*07/08/2019*)
    - Clear and detailed instructions with the short clip on instructions e.g. dragging/drop files into histories. Awesome way to learn! (*09/08/2019*)
    - User friendly (*14/08/2019*)
    - Simple and clear (*14/08/2019*)
    - thorough (*19/08/2019*)
    - good explanations (*20/08/2019*)
    - The enogth for use in my new job (*20/08/2019*)
    - The step by step teaching (*01/09/2019*)
    - tutorial easy to follow (*06/09/2019*)
    - So mouch (*06/09/2019*)
    - The simplicity and practicality of the tutorial (*11/09/2019*)
    - The step-by-step guidance with explanations. (*11/09/2019*)
    - informative (*12/09/2019*)
  - Con:
    - the clear instruction pattern (*09/09/2018*)
    - Nothing (*08/10/2018*)
    - some buttons do not exist (anymore)? Like in "Look at all your histories" under 3. at the top line there is a "done" button - this was not displayed when I did it. That is not a tragedy but it can be confusing.... (*10/10/2018*)
    - speed (*17/10/2018*)
    - none (*18/10/2018*)
    - There is no answer to the question under the section entitled 'Re-run that tool with changed settings'. A 'Done' button does not appear in the view history screen as well.  (*18/10/2018*)
    - - (*23/10/2018*)
    - You could link to short description of what a fastq format is, etc... and warn that Filterbyquality sometimes is not present. I could not find it. I use FilterFASTQ instead (*29/10/2018*)
    - Nothing (*30/10/2018*)
    - This should be the first tutorial that shows up, even before the longer version as now I would feel more comfortable doing the long one whereas that one was overwhelming at first. (*01/11/2018*)
    - Nothing (*19/11/2018*)
    - everything is ok (*05/01/2019*)
    - I found the boxes around each of the big steps a little distracting. They are not necessary for the transition between the steps. The large headings are enough. (*23/01/2019*)
    - more detail about data set  (*05/02/2019*)
    - Nothing really,  (*12/02/2019*)
    - niks  (*13/02/2019*)
    - explain how to upload data from different sources (*20/02/2019*)
    -  some video lectures of the tutorial may be added  (*07/03/2019*)
    - it would be great if the quality score part is explained more. (*14/03/2019*)
    - Include in the tutorial that you have to validate your email address if you a new galaxy user who just registered. (*18/03/2019*)
    - If the tutorials could be downloadable, it could be a great help (*16/04/2019*)
    - If you make a question, include always the answer with an appropiate explanation to make everything clear. Why in the case you use the 36 filter there are so many reads discarded? Do not Forget most of people is the first time they work with this online tool.   (*22/04/2019*)
    - None (*22/04/2019*)
    - Needs to be updated. I could not follow the tutorial because the tools, the interface, etc. are different (*05/06/2019*)
    - the Filter by quality function was not on the program (*14/06/2019*)
    - none (*21/06/2019*)
    - will not know until I work on my own data (*23/06/2019*)
    - The history names in fig 1 could be showing as the next-analysis and my-analysis. (*28/07/2019*)
    - provide more links so we could be able to try and check if we understand galaxy certainly (*05/08/2019*)
    - NA -thought this was excellent and easy to follow! (*09/08/2019*)
    - multiple ways of importing data in the introduction (*14/08/2019*)
    - In this moment I dont have a suggest maybe with the practiceses (*20/08/2019*)
    - More publicity to make this fantastic site more popular to bioinformatic field (*01/09/2019*)
    - didn't felt need for more info (*06/09/2019*)
    - . (*06/09/2019*)
    - More exploration of the data analysis to aid those who are not familiar with biological data (e.g. brief explanation of FASTQ and how to interpret it) (*11/09/2019*)
    - nothing (*12/09/2019*)

- **From peaks to genes**
  - Pro:
    - Well structured, easy to understand. (*17/09/2018*)
    - pink note (*17/09/2018*)
    - clear procedure, good for beginners (*17/09/2018*)
    - well explained (*17/09/2018*)
    - Well-structured and helpful (*17/09/2018*)
    - easy to follow instructions with images (*27/09/2018*)
    - easy to follow instructions with images (*27/09/2018*)
    - simplicity (*02/10/2018*)
    - scope,  topic and clarity of tutorial (*17/10/2018*)
    - very clear and detailed instructions (*17/10/2018*)
    - it is clear (*17/10/2018*)
    - Nice clear , very informative (*17/10/2018*)
    - the structure that allows you to actually do the work.; clear tutorial and good help desk by walking around (*17/10/2018*)
    - Easy setting, sufficient help during the event of the organizers,  (*17/10/2018*)
    - easy to follow individual steps (*22/11/2018*)
    - Easy to follow tutorial (*22/11/2018*)
    - The tutorial. (*20/12/2018*)
    - Step by step introduction and repetition of features (*08/01/2019*)
    - Very clear instruction and step-by-step explanation (*25/02/2019*)
    - It is easy to follow (*25/02/2019*)
    - use of different tools, good explanations (*25/02/2019*)
    - most but if the files that need to be downloaded do not exist at UCSC, this tutorial is useless (*05/03/2019*)
    - short calculation times (*11/06/2019*)
    - I didn't have the time to like it since the intersect command is not working :( (*15/08/2019*)
    - not working with the bedtools intersect intervals (the other tool is currently not working) (*21/08/2019*)
  - Con:
    - maybe by adding more pictures next to the explanations? (*17/09/2018*)
    - More information on the flanking region tool (how it works) (*17/09/2018*)
    - pick a relevant induction based on your needs  (*02/10/2018*)
    - More visual examples (*08/10/2018*)
    - would like to have example outputs for important steps, so if i made a mistake in step 7 i don't have to find out in step 19 and go back to revise everything. I can just use the dummy dataset for step 19 to continue and finish the practice (*17/10/2018*)
    - airconditioning less cold (*17/10/2018*)
    - explaning more explicitly the goal of each individual exercise (*17/10/2018*)
    - short of time to complete the excercise (*17/10/2018*)
    - Chipseq data seems a little more complex than more 1-dimensional seq reads (DNA/RNAseq), so would seem to me that starting with e.g. RNAseq analysis could be more logical  (*22/11/2018*)
    - We did not realize the 00000000rik names were also gene names, so we spent some time thinking we did something wrong... (*22/11/2018*)
    - More practical excercises (*20/12/2018*)
    - some small exercises or quizes (*25/02/2019*)
    - go through the process and correct (*05/03/2019*)
    - I would love it to work, the tools changed in the meanwhile and it need to be updated cause it is not working anymore for some not clear reason . Most probably there is a problem with the UCSC file (I changed any kind of parameter on the interval file and i get always the same error). (*15/08/2019*)
    - Update (*21/08/2019*)

- **Galaxy 101**
  - Pro:
    - very detailed and easy to follow, even for a complete beginner - great! (*14/09/2018*)
    - That all the tools had helpful explanations (*15/09/2018*)
    - The presentation of the tutorial and the practicality of it (*26/09/2018*)
    - I found it interesting and user friendly (*27/09/2018*)
    - It was easy to comprehend the content (*27/09/2018*)
    - easy to follow (*27/09/2018*)
    - Very easy to follow, well explained (*31/10/2018*)
    - Easy to follow (*28/11/2018*)
    - Easy to follow, but the task is not trivial! (*28/11/2018*)
    - Very informative, and nicelly (*07/12/2018*)
    - Very good explanations, good demonstration of Galaxy potential for begginer  (*31/01/2019*)
    - Screenshots showing which options should be selected  (*04/02/2019*)
    - The step are easy to follow  (*10/02/2019*)
    - Very simple to follow and learn on my own (*25/02/2019*)
    - Very straight-forward. Easy reading, easy learning. Gives a great first contact with the software. (*13/03/2019*)
    - The step by step demonstration was really helpful (*03/04/2019*)
    - I liked being able to compare my screenshots and felt the directions were generally very good. Definitely have a better handle on how to use the program, especially re-using workflows. (*25/04/2019*)
    - explanations were very clear ! (*25/04/2019*)
    - Instructive (*01/05/2019*)
    - # No results returned from query. (*15/05/2019*)
    - Ease of understanding (*16/05/2019*)
    - Very detailed description of every step (*29/05/2019*)
    - The instructions were very clear and easy to follow (*30/05/2019*)
    - Nice combination of presentations and tutorial, clear steps including tips and comments (*11/06/2019*)
    - Good introduction to Galaxy. I like the worklflow function the most.  (*11/06/2019*)
    - Good speed, detailed explanation for the introduction (*11/06/2019*)
    - It explains really clearly. (*16/06/2019*)
    - its very helpful to get a basic idea of how the platform works (*06/07/2019*)
    - I liked how easily it flows, some concepts of biology are briefly explained, it helps a lot!  (*02/08/2019*)
    - Screenshots and clear colour coding made it extremely easy to follow (*28/08/2019*)
    - It was very clear and intuitive  (*28/08/2019*)
    - it was sufficient (*28/08/2019*)
    - Easy to follow and useful (*28/08/2019*)
    - Very clear (*28/08/2019*)
    - Very easy to follow (*06/09/2019*)
    - very detailed (*10/09/2019*)
    - It was really easy to follow and interesting to learn (*12/09/2019*)
  - Con:
    - If there was a guide to say which tool to use for a specific analysis to be made. Unless I know which tool to look for, I'd be lost (*15/09/2018*)
    - The rate of update of the tutorial (*26/09/2018*)
    - nothing for now (*27/09/2018*)
    - Nothing (*31/10/2018*)
    - One could add a section describing the use of multiple datasets, tags, etc. (*04/11/2018*)
    - Including some assignment or home work (*28/11/2018*)
    - update some of the options and screenshots to be consistent with the latest version of Galaxy (*28/11/2018*)
    - The explanation about the use workflow on Repeats (*07/12/2018*)
    - Galaxy at usegalaxy.org ends up showing "the white screen of death" extremely often (10s of times during following this tutorial). (*14/01/2019*)
    - Should provide practice every step (*10/02/2019*)
    - Fatal error: Exit code 1 () Error running sorter.py Command '(grep '^#' /galaxy-repl/main/files/030/998/dataset_30998511.dat) >> /galaxy-repl/main/files/030/998/dataset_30998729.dat' returned non-zero exit status 1 (*18/04/2019*)
    - Some of the instructions are outdated as newer versions of data in UCSC main have been added, which makes some parts slightly confusing (*25/04/2019*)
    - this is a bit too long (*25/04/2019*)
    - I do not know - I know only Biopython (*01/05/2019*)
    - # No results returned from query. (*15/05/2019*)
    - Adding more examples of its applications and how it is used in real time (*16/05/2019*)
    - I couldn't find a 'display at UCSC browser' option when I get to this step (*29/05/2019*)
    - It would be nice if there is more explanations on the significance of each step and what exactly is being performed when selecting different tools. (*30/05/2019*)
    - CouldnÂ´t really see what the results in the genome browser would tell me and how I apply this for my work.  (*11/06/2019*)
    - by giving more free tutorials (*06/07/2019*)
    - Maybe add more examples, more explanations on the datasets used.  (*02/08/2019*)
    - More clarity in instructions for the solution to the question; "For the first 3 exons in your file, what is the number of SNPs that fall into that exon?" The description of the solution steps was unclear. (*09/08/2019*)
    - More clarity in instructions for the solution to the question; "For the first 3 exons in your file, what is the number of SNPs that fall into that exon?" The description of the solution steps was unclear. (*09/08/2019*)
    - Nothing - I thought it was great! :) (*28/08/2019*)
    - Nothing (*28/08/2019*)
    - the tiny mistakes (*28/08/2019*)
    - Ability to zoom on the graphics (*06/09/2019*)
    - none (*10/09/2019*)
    - Its good (*12/09/2019*)

- **IGV Introduction**
  - Pro:
    - That there was a tutorial to begin with!   Thank you! (*05/06/2019*)
  - Con:
    - Actually show some use-cases (there is only one and i am unable to load ENCODE data) (*15/01/2019*)
    - Cannot find C2C12 file (*25/02/2019*)
    - One cannot explain an unknown by substituting another unknown.   Therefore do not use abbreviations such as BAM, BED, MAF, VCF etc.  Consider incorporating a complete or partial glossary (*05/06/2019*)
    - RNASeq of C2C12 not available (*22/08/2019*)

- **Introduction to Genomics and Galaxy**
  - Pro:
    - very detailed and easy to follow, thank you (*15/09/2018*)
    - Broke down the concepts well. Introduced ideas one  step at a time with good explanations that minimized jargon. And worked to make concepts tangible with hands on use of Galaxy. (*28/02/2019*)
    - very clear and easy to follow (*09/04/2019*)
  - Con:
    - Galaxy says it has ran the workflow, but nothing happens. (*16/01/2019*)
    - It could be better to put the overlapped extron images here too (*09/04/2019*)
    - intersect never worked (*22/08/2019*)
    - Update the intersect phase since that tool is no more available (*23/08/2019*)

- **NGS data logistics**
  - Pro:
    - good overview over method and files (*16/09/2018*)
    - good overview over method and files (*16/09/2018*)
    - good overview over method and files (*16/09/2018*)
    - makes life easy (*27/09/2018*)
    - a very well written tutorial with detailed information for a person learning Chip-Seq from scartch  (*03/11/2018*)
    - Clear and consise (*17/11/2018*)
    - The explanation is very clear and concise, thanks! (*27/11/2018*)
    - Explanation of theoretical basics. Links to additional resources. (*17/01/2019*)
    - awesome and answered many of the questions in my head (*22/01/2019*)
    - I liked the videos. (*30/04/2019*)
    - the explanation part (*23/05/2019*)
    - all of it (*24/05/2019*)
    - Anyone could have understood what it has been explained!! (*03/06/2019*)
  - Con:
    - not as detailed and easy to follow as the first tutorials, examples not well shown (*16/09/2018*)
    - not as detailed and easy to follow as the first tutorials, examples not well shown (*16/09/2018*)
    - not as detailed and easy to follow as the first tutorials, examples not well shown (*16/09/2018*)
    - very good (*27/09/2018*)
    - I don't see the files name as sample1-r.fq.gz to sample 4 to download and do that practically. However, I do understand the video and it was very helpful (*03/11/2018*)
    - Mark Duplicates step fails:  Fatal error: Exit code 1 () Picked up _JAVA_OPTIONS: -Djava.io.tmpdir=/galaxy-repl/main/jobdir/022/055/22055178/_job_tmp -Xmx7g  -Xms256m 11:37:37.876 INFO  NativeLibraryLoader - Loading libgkl_compression.so from jar:file:/cvmfs/main.galaxyproject.org/deps/_conda/pkgs/picard-2.18.2-py36_0/share/picard-2.18.2-0/picard.jar!/com/intel/gkl/native/libgkl_compression.so [Thu Jan 17 11:37:37 CST 2019] MarkDuplicates INPUT=[Filter_on_data_7__Filtered_BAM] OUTPUT=/galaxy-repl/main/files/029/341/dataset_29341054.dat METRICS_FILE=/galaxy-repl/main/files/029/341/dataset_29341053.dat REMOVE_DUPLICATES=true ASSUME_SORTED=true DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES OPTICAL_DUPLICATE_PIXEL_DISTANCE=100    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true ADD_PG_TAG_TO_READS=true PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false [Thu Jan 17 11:37:37 CST 2019] Executing as g2main@roundup55.tacc.utexas.edu on Linux 3.10.0-693.21.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_152-release-1056-b12; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.18.2-SNAPSHOT INFO 2019-01-17 11:37:38 MarkDuplicates Start of doWork freeMemory: 248936480; totalMemory: 259588096; maxMemory: 7265714176 INFO 2019-01-17 11:37:38 MarkDuplicates Reading input file and constructing read end information. INFO 2019-01-17 11:37:38 MarkDuplicates Will retain up to 26325051 data points before spilling to disk. [Thu Jan 17 11:37:38 CST 2019] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 0.02 minutes. Runtime.totalMemory()=382373888 To get help, see http://broadinstitute.github.io/picard/index.html#GettingHelp Exception in thread "main" htsjdk.samtools.SAMFormatException: SAM validation error: ERROR: Record 262, Read name M01368:16:000000000-A3M99:1:2114:3862:13382, bin field of BAM record does not equal value computed based on alignment start and end, and length of sequence to which read is aligned  at htsjdk.samtools.SAMUtils.processValidationErrors(SAMUtils.java:454)  at htsjdk.samtools.BAMFileReader$BAMFileIterator.advance(BAMFileReader.java:812)  at htsjdk.samtools.BAMFileReader$BAMFileIterator.next(BAMFileReader.java:797)  at htsjdk.samtools.BAMFileReader$BAMFileIterator.next(BAMFileReader.java:765)  at htsjdk.samtools.SamReader$AssertingIterator.next(SamReader.java:576)  at htsjdk.samtools.SamReader$AssertingIterator.next(SamReader.java:548)  at picard.sam.markduplicates.MarkDuplicates.buildSortedReadEndLists(MarkDuplicates.java:495)  at picard.sam.markduplicates.MarkDuplicates.doWork(MarkDuplicates.java:232)  at picard.cmdline.CommandLineProgram.instanceMain(CommandLineProgram.java:282)  at picard.cmdline.PicardCommandLine.instanceMain(PicardCommandLine.java:98)  at picard.cmdline.PicardCommandLine.main(PicardCommandLine.java:108) (*17/01/2019*)
    - nothing I can think of (*22/01/2019*)
    - I would suggest to make the videos longer and theory shorter. (*30/04/2019*)
    - it is nice (*23/05/2019*)
    - add example with bowtie2  (*24/05/2019*)
    - Please date this content.  Is it out of date?  It does not mention kallisto, sleuth or salmon.  Are these available in a cloud somewhere?  Are they better? (*27/08/2019*)

