- **Mapping**
  - Pro:
    - Very detailed, thank you (*2018-09-17 00:00:00*)
    - Easy to follow, great questions, great explanations! (*2018-12-18 00:00:00*)
    - Very clear instructions  (*2019-07-15 00:00:00*)
    - Almost everything (*2019-07-18 00:00:00*)
    - The Questions (*2019-10-09 00:00:00*)
    - yes. thank you :) (*2019-11-30 00:00:00*)
    - It was largely well-written. (*2020-01-08 00:00:00*)
    - the methodology (*2020-01-17 00:00:00*)
    - Everything (*2020-03-03 00:00:00*)
    - Easy to follow for beginners (*2020-04-08 00:00:00*)
    - Es  preciso en las indicaciones.  (*2020-05-30 00:00:00*)
  - Con:
    - More details on the Selection of analysis mode (non default option) (*2019-07-15 00:00:00*)
    - In step 1 of "Hands-on: Mapping with Bowtie2" your instructions mention that the paired-end input is the output of Trim Galore!. However, the previous tutorial on quality control uses MultiQC. As such, the reader of this tutorial will be confused.  In "Hands on-Inspect a BAM/SAM file" the 11 fields listed in the tutorial (QNAME, FLAG...) do not match with the 10 field names in the output produced (QNAME, FLAG, RNAME, POS, MAPQ, CIGAR, MRNM, MPOS, ISIZE, SEQ). The JBrowse viewer, doesn't work for me, get an error (*2019-07-18 00:00:00*)
    - More detailed information, hard to follow instructions (*2019-11-06 00:00:00*)
    - Images of Alignment using GATK or BWA in IGV (*2019-11-30 00:00:00*)
    - The JBrowse tool would not show me anything in that bit of the chromosome and kept erroring 'name can't be found' (*2020-01-08 00:00:00*)
    - Explain the interest of ALL the options for the software  (*2020-01-17 00:00:00*)
    - More detailed explanation might be helpful (*2020-03-03 00:00:00*)
    - Need more information about non default setting of Bowtie2 and SAMtools (*2020-04-08 00:00:00*)
    - es poco ilustrativo y debería explicar con mas detalle la parte de la inserción de los datos mediante biblioteca. (*2020-05-30 00:00:00*)
    - Very informative tutorial to gain practical experience mapping reads on reference genome. This tutorial have cleared many key points regarding NGS datasets cleaning, filtering and mapping the reads on reference genome.  (*2020-06-02 00:00:00*)

- **Quality Control**
  - Pro:
    - very detailed, easy to follow, thank you (*2018-09-17 00:00:00*)
    - step by step guideline (*2018-10-07 00:00:00*)
    - Examples and questionaries with solutions :-) (*2018-10-08 00:00:00*)
    - all of it! (*2018-10-31 00:00:00*)
    - Relatively clear to work through (*2018-11-28 00:00:00*)
    - step-by-step (*2018-12-01 00:00:00*)
    - Nice explanation, really easy! (*2018-12-10 00:00:00*)
    - all tool available single platform   (*2018-12-27 00:00:00*)
    - The instructions were really clear (*2019-01-25 00:00:00*)
    - very clear (*2019-03-20 00:00:00*)
    - It's clear enough, and contain the basic but necessary information for RNA-seq raw data Quality Control analysis. (*2019-04-24 00:00:00*)
    - this is so clear !  (*2019-05-13 00:00:00*)
    - solid logic and small amount of valuable information (*2019-05-14 00:00:00*)
    - I liked the most the part in which you explained the algorithm for the cutadapt tool, the concrete example was fundamental to understand everything clearly! (*2019-06-04 00:00:00*)
    - step by step QC analysis (*2019-06-11 00:00:00*)
    - Very Clear and well-thought instructions. Appreciated effort  (*2019-07-15 00:00:00*)
    - Amost everything! (*2019-07-17 00:00:00*)
    - Step-by-step instructions and showing results. Questions to test understanding. (*2019-07-18 00:00:00*)
    - step by step instruction (*2019-07-27 00:00:00*)
    - Way of explanation. (*2019-08-15 00:00:00*)
    - Easy to follow steps (*2019-08-21 00:00:00*)
    - The material (*2019-08-23 00:00:00*)
    - Preciseness (*2019-09-04 00:00:00*)
    - really helpful for first timers using fastqc, thank you. (*2019-09-14 00:00:00*)
    - Good advices by the teacher to understand the workflow (*2019-09-24 00:00:00*)
    - easy to follow (*2019-10-07 00:00:00*)
    - speed was good (*2019-10-07 00:00:00*)
    - Very explanatory tutorial (*2019-10-07 00:00:00*)
    - Good overview of the principle (*2019-10-07 00:00:00*)
    - there was a lot of examples  (*2019-10-07 00:00:00*)
    - simplicity of the tutorial (*2019-10-07 00:00:00*)
    - Tutorial is very detailed and clear (*2019-10-07 00:00:00*)
    - the dataset andexamples (*2019-10-07 00:00:00*)
    - I like that there is a lot of info and examples.  (*2019-10-07 00:00:00*)
    - Materials are well crafted. (*2019-10-07 00:00:00*)
    - Overall, the manual and following the course was good. (*2019-10-07 00:00:00*)
    - Step sise instructions  (*2019-10-08 00:00:00*)
    - Easy to follow (*2019-10-11 00:00:00*)
    - easy to operate and more information extract easily on one click (*2019-12-22 00:00:00*)
    - It was so clearly divided up. I loved the different icons/open/close boxes that made things make so much sense AND could show main points & extra info. It's hard as an outsider to understand priorities, so that was brilliant. (*2020-01-07 00:00:00*)
    - It is very interesting tutorial (*2020-01-13 00:00:00*)
    - That I can get extra info about some of the things, but it is not included in the main scope of the tutorial (the grey info boxes). For my purpose right now, I want to read it, so I am gratefull it is there. But I also sometimes check manuals where I don't need to know everything, but just get my analysis to work (*2020-03-26 00:00:00*)
    - The tutorial is clear and easy to be understand (*2020-04-09 00:00:00*)
    - I liked to explain the steps thank you (*2020-04-15 00:00:00*)
    - I liked examples that I could use for practice. (*2020-05-18 00:00:00*)
    - Clarity  (*2020-05-20 00:00:00*)
    - Question and answers + hands on training (*2020-05-23 00:00:00*)
    - I understand how to access and examine the raw sequence data and filtering and trimming process on Galaxy to increase the sequence quality of reads for downstream analysis.  (*2020-06-01 00:00:00*)
  - Con:
    - the answers that are given to questions could be a bit more elaborate and give a/or link to a more detailed explaination (*2018-09-17 00:00:00*)
    - Not clear that Trim Galore and FastQ are integrated into Galaxy. Uncertain how obtained 97,644 sequences after trim galore of bad data (I got 99,341). (*2018-09-22 00:00:00*)
    - the webpage output for MultiQC is not functioning (*2018-10-07 00:00:00*)
    - More details regarding others output sections of FastQC tool (*2018-10-08 00:00:00*)
    - how to rmemeber steps.... (*2018-10-31 00:00:00*)
    - Doesn't need to be (*2018-11-28 00:00:00*)
    - often ambiguous (*2018-12-01 00:00:00*)
    - what is the next step? (*2018-12-10 00:00:00*)
    - more precise knowledge like FASTQC is all data copy and paste from their original source need more easy way explain of documentation    (*2018-12-27 00:00:00*)
    - MultiQC didn't work (*2019-01-25 00:00:00*)
    - specificity of quality check regarding the type of data (16S, whole genome) (*2019-03-20 00:00:00*)
    - suggestion of several programs that can be used eg for trimming (Trim Galore...) (*2019-04-08 00:00:00*)
    - You could including how to do FastQC scripts (*2019-04-24 00:00:00*)
    - Probably as we are not trimming any adaptors in this tutorial, Trimmomatic can also be used on datasets provided. (*2019-05-14 00:00:00*)
    - quality of audio in presentation (*2019-06-11 00:00:00*)
    - In your tutorial, you state "In this training, no adapters were already removed". What does this mean? That adapters were not removed? Then why do we not remove them in the tutorial? From a grammatical point of view your sentence is confusing. Do you mean "In this training, adapters were not yet removed"? (*2019-07-17 00:00:00*)
    - don't know (*2019-07-27 00:00:00*)
    - Examples can be included.  (*2019-08-15 00:00:00*)
    - Considering SNP in the analysis, practice adapters removal ?  (*2019-08-21 00:00:00*)
    - Figuring out why the mapping tutorial directed me here to quality control fastq files then they will not work... (*2019-08-23 00:00:00*)
    - More examples (*2019-09-04 00:00:00*)
    - provide images of what the results should look like. Also, there are a few typos like 'how many basepairs has been removed' instead of 'how many base-pairs have been removed' (*2019-09-14 00:00:00*)
    - I was already familiar with fastqc and cutadapt so I don't know (*2019-10-07 00:00:00*)
    - The available paremeters per step should be more elaborated in order to understand how we can use them in other context and queries (*2019-10-07 00:00:00*)
    - More detailed explanations regarding the utilities of each available tool (*2019-10-07 00:00:00*)
    - at some points sound is not good (*2019-10-07 00:00:00*)
    - Use more examples (eg for problematic datasets) (*2019-10-07 00:00:00*)
    - To show more of the bad quality examples (*2019-10-07 00:00:00*)
    - for me it was not clear when you use this tools for example multi qc and what are the results (*2019-10-07 00:00:00*)
    - explain more the available options of each menu and the biological meaning of each choice (*2019-10-07 00:00:00*)
    - Please wait a little bit each time you act on the website (*2019-10-07 00:00:00*)
    - Sound issues. Need of a better microphone? Could also be the speakers in Estonia. (*2019-10-07 00:00:00*)
    - More explanation and better voice quality. (*2019-10-07 00:00:00*)
    - Provide example data where 3-4 paired end samples provided. Especially, how to analyse and visualize QC data for multiplex samples.  (*2019-10-08 00:00:00*)
    -  Not much really (*2019-10-11 00:00:00*)
    - why not using trim galore instead of cutadapt? (*2019-11-20 00:00:00*)
    - noun (*2019-12-22 00:00:00*)
    - everything is great :) (*2020-04-09 00:00:00*)
    - I think everything was good for me. But I gave you 4-on-5 because I had a hard time applying the last two phases. (*2020-04-15 00:00:00*)
    - I would like to see more information about adapter detection and removal. (*2020-05-18 00:00:00*)
    - explaination of all the inputs of fastqc and cutadapt (*2020-05-23 00:00:00*)

